webapp/src/components/BuyPage/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { Item } from '@dcl/schemas'
+import { toBN } from 'web3x/utils'
+import { Asset, AssetType } from '../../modules/asset/types'
+
+/**
+ * Checks against the min value of costless item sales
+ */
+export function isPriceTooLow(type: AssetType, asset: Asset) {
+  if (type === AssetType.ITEM) {
+    const { price } = asset as Item
+    const minSaleValue = getMinSaleValueInWei()
+    if (minSaleValue) {
+      return toBN(price).gte(toBN(minSaleValue))
```

Comment:
Isn't it lower than?
```suggestion
      return toBN(price).lt(toBN(minSaleValue))
```

Reply:
Correct, the function is actually fine, the refactored name was the problem. Changing tho!